department-of-veterans-affairs/LEAF
# LEAF-4030: Multi Select Options on Email Templates not formatted correctly

Implemented update/fix to have multiselect fields show in emails formatted properly and in basic bullets.

### Potential Impact
Emails sent with custom fields now show properly formatted multiselect field values.

### Testing
- Navigate to the Email Template Editor
- Update custom email template to include a multiselect field
- Navigate to Workflow Editor
- Alter action in designated workflow to include the custom email template being used to send emails
- Navigate to Form Editor
- Verify form is associated with above workflow
- Create new request using above form
- Submit the form and activate the above action
- Verify the field is formatted correctly in the sent email
